Feature/single shard autoincrement key (#1013)
arangodb/docs: 3.10/architecture-single-instance-vs-cluster.md

Document Keys
-------------

In a cluster the *autoincrement* key generator is supported for single-sharded

> collections. Otherwise, for collections with more than one shard, you would 
> have to use the *traditional* or user defined keys.

Reviewer: ```suggestion
collections. It cannot be used for collections with more than one shard.
```
